calabash-cucumber/lib/calabash-cucumber/launcher.rb

```diff
@@ -290,21 +290,29 @@ def reset_app_sandbox(opts={})
 })
   end
 
-  # Erases the contents and setting for every available simulator.
-  #
-  # For Xcode 6, this is equivalent to calling: `$ xcrun simctl erase` on
-  # every available simulator.  For Xcode < 6, it is equivalent to touching
-  # the 'Reset Content & Settings' menu item.
+  # Erases a simulator. This is the same as touching the Simulator
+  # "Reset Content & Settings" menu item.
   #
+  # @param [RunLoop::Device or UDID or human readable name] The simulator to erase.
   # @note
-  #  **WARNING** This is a destructive operation.  You have been warned.
-  #
-  # @raise RuntimeError if called when targeting a physical device
-  def reset_simulator
+  #   If no device given every available simulator will be erased.
+
+  # @raise ArgumentError If the simulator is a physical device
+  # @raise RuntimeError If the simulator cannot be shutdown
+  # @raise RuntimeError If the simulator cannot be erased
+
+  def reset_simulator(device=nil)
     if device_target?
       raise "Calling 'reset_simulator' when targeting a device is not allowed"
+    else
+      if device.nil? || device.empty?
```

Comment:
Not sure, because default simulator not always used as a default :)
i mean i have default "iPhone 6s (9.1)" but mainly override that with DEVICE_TARGET 
from my point of view we can extend it by checking if device == default and in that case erase default one